07_errors/kasunfdo/error.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import "fmt"
+
+const (
+	ErrInvalidVal = 400
+	ErrNotFound   = 404
+	ErrInternal   = 500
+	ErrOpenDB     = 501
+)
+
+var ErrStr = map[int]string{
+	ErrInvalidVal: "value cannot be negative",
+	ErrNotFound:   "puppy with id=%v not found",
+	ErrInternal:   "internal system error",
+	ErrOpenDB:     "error occurred while opening db %v",
```

Comment:
fold it into `ErrInternal` or create `ErrSetup` IMO

Reply:
Agreed.